packages/code-du-travail-data/dataset/themes/query.js

```diff
@@ -0,0 +1,26 @@
+const themes = require("./themes.json");
+const memoizee = require("memoizee");
```

Comment:
I would have called it `memoize` but I don't care ^^

Reply:
on the other hand we know that's the memoize function from a 3rd party lib ;)